src/main.cpp

```diff
@@ -490,6 +492,12 @@ int init_options(int argc, char**argv, struct options& options)
     }
   }
 
+  if (!diameter_timeout_set)
```

Comment:
I considered this, but at that point I need to make `diameter_timeout_set` global or pass it into `init_options()`, neither of which is very nice. What do you think?